CHANGELOG.md

```diff
@@ -185,6 +185,8 @@
   * Remove changed property name from export.
     FIXES [#5300](https://github.com/microsoft/Microsoft365DSC/issues/5300)* IntuneSecurityBaselineMicrosoftEdge
   * Deprecate property `authschemes` and replace with `AuthSchemes_AuthSchemes`
+* IntuneVPNConfigurationPolicyIOS
```

Comment:
Please put this entry in the Unreleased section at the top of the file.

Reply:
Haha, if there's a way to mess it up, I'll find it... I've moved it to Unreleased section. Thanks!